app/assets/javascripts/backbone/views/timeline_batch_actions_view.coffee

```diff
@@ -36,7 +36,7 @@ class MS.Views.TimelineBatchActionsView extends Backbone.View
     dependents.each ->
       stepId = $(this).data('id')
       precedentId = $(this).data('precedent-id')
-      unselectedPrecedentIds.push(precedentId) unless stepIds.includes(precedentId)
+      unselectedPrecedentIds.push(precedentId) unless stepIds.indexOf(precedentId) != -1
```

Comment:
I just wrote a big comment about this in the other PR. I still think there is a tricky bug hiding in here. Take a look at the other comment.